install/tools/istio_vm_common.sh

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env bash
+
+# Helper functions for extending the cluster with external VMs.
+
+# Script can be sourced in other files or used from tools like ansible.
+# Environment variables used:
+
+# ISTIO_FILES - directory where istio artifacts are downloaded, default to current dir
+
+# Initialize internal load balancers to access K8S DNS and Istio Pilot, Mixer, CA.
+# Must be run once per cluster.
```

Comment:
could we document dependencies and params for this helper script?

Reply:
Done.